README.md

```diff
@@ -10,7 +10,7 @@
 
 Welcome to our little coding exercise. During this test you will be given the opportunity to play with JavaScript, React and rockets in the same project.
 
-We recommend you to block a time slot of 2-3 hours to complete the exercise, but feel free to accommodate your time as needed. Also, before you begin **please make sure that you have read and understood this readme file entirely**. If there's anything that you don't understand, please don't hesitate to reach out and ask.
+We recommend you to block a time slot of 2-3 hours to complete the exercise, but feel free to accommodate your time as needed. Also, before you begin **please make sure that you have read and understood this readme file entirely**. If there's anything that you don't understand, please don't hesitate to reach out and ask. For that, you can create a GitHub issue in your private clone repo, describe your question(s) and mention users from the "collaborators" list below, we will be notified about your question and reach out to clarify shortly. 
```

Comment:
Maybe we should include this as a separate section just before "Helpful links" section, what do you think?